test/Dialect/Check/check-ops-fail.mlir

```diff
@@ -30,3 +36,25 @@ func.func @myfunc2() {
   check.expect_almost_eq(%a, %b, %threshold): tensor<4x4xf32>, tensor<4x4xf32>, f32
   return
 }
+
+// -----
+// REDEFINE: %{option} = entry3
+// RUN: %{command}
+
+func.func @entry3(%arg0:tensor<4x4xf32>){
+  check.expect_sane(%arg0): tensor<4x4xf32>
```

Comment:
this may have unexpected results if the memory happens to have "reasonable" float values.

Reply:
Fair enough, removing this test.